easybuild/tools/options.py

```diff
@@ -1451,17 +1452,22 @@ def set_up_configuration(args=None, logfile=None, testing=False, silent=False):
     init_build_options(build_options=build_options, cmdline_options=options)
 
     # done here instead of in _postprocess_include because github integration requires build_options to be initialized
-    if eb_go.options.include_easyblocks_from_pr:
-        easyblocks_from_pr = fetch_easyblocks_from_pr(eb_go.options.include_easyblocks_from_pr)
+    pr_easyblocks = eb_go.options.include_easyblocks_from_pr
+    if pr_easyblocks:
+        easyblocks_from_pr = fetch_easyblocks_from_pr(pr_easyblocks)
+        included_from_pr = set([os.path.basename(eb) for eb in easyblocks_from_pr])
 
         if eb_go.options.include_easyblocks:
-            # make sure we're not including the same easyblock twice
-            included_from_pr = set([os.path.basename(eb) for eb in easyblocks_from_pr])
-            included_from_file = set([os.path.basename(eb) for eb in eb_go.options.include_easyblocks])
+            # check if you are including the same easyblock twice
+            included_paths = expand_glob_paths(eb_go.options.include_easyblocks)
+            included_from_file = set([os.path.basename(eb) for eb in included_paths])
             included_twice = included_from_pr & included_from_file
             if included_twice:
-                raise EasyBuildError("Multiple inclusion of %s, check your --include-easyblocks options",
-                                     ','.join(included_twice))
+                print_warning("EasyBlock(s) %s included from multiple locations, the one from a PR will be used" %
+                              ','.join(included_twice))
```

Comment:
@migueldiascosta We can mention the PR # in the warning?

```suggestion
                warning_msg = "One or more easyblocks included from multiple locations: %s " % ', '.join(included_twice)
                warning_msg += "(the one(s) from PR #%s will be used)" % pr_easyblocks
                print_warning(warning_msg)
```